packages/gatsby-theme-newrelic/src/components/Footer.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from './Icon';
+import Logo from './Logo';
+import ExternalLink from './ExternalLink';
+import { graphql, useStaticQuery, Link } from 'gatsby';
+import { css } from '@emotion/core';
+
+const Footer = ({ fileRelativePath }) => {
```

Comment:
[nit] Should we call this `GlobalFooter` since we called the main header the `GlobalHeader`? It might also better signal that this isn't just a generic footer element, but should be used as the site's main footer. Thoughts?

Reply:
I went back and forth on this, since I feel like it's "global" because it's coming from the theme. To stay consistent, I'm happy to adjust this to match.

Reply:
If we wanted to stick with the `Footer` name, would it make sense to rename `GlobalHeader` to `Header` to match?